pyaedt/hfss.py

```diff
@@ -1935,39 +1806,42 @@ def create_spiral_lumped_port(self, start_object, end_object, port_width=None):
             xsection_type="Line",
             xsection_orient=orient,
             xsection_width=closest_distance / 2,
-            name=end_object + "_sheet",
+            name=reference + "_sheet",
         )
 
         # assign pec to created polylines
-        self.assign_perfecte_to_sheets(poly1, sourcename=start_object)
-        self.assign_perfecte_to_sheets(poly2, sourcename=end_object)
+        self.assign_perfecte_to_sheets(poly1, name=assignment)
+        self.assign_perfecte_to_sheets(poly2, name=reference)
 
         # create lumped port on spiral
         port = self.lumped_port(spiral, reference=[poly2.name], name=name)
 
         return port
 
-    @pyaedt_function_handler()
-    def create_voltage_source_from_objects(self, startobj, endobject, axisdir=0, sourcename=None, source_on_plane=True):
+    @pyaedt_function_handler(startobj="assignment", endobject="reference", sourcename="name", axisdir="start_direction")
+    def create_voltage_source_from_objects(
+        self, assignment, reference, start_direction=0, name=None, source_on_plane=True
+    ):
         """Create a voltage source taking the closest edges of two objects.
 
         Parameters
         ----------
-        startobj :
-            Starting object for the integration line.
-        endobject :
-            Ending object for the integration line.
-        axisdir : int or :class:`pyaedt.application.Analysis.Analysis.AxisDir`, optional
-            Position of the port. It should be one of the values for
-            ``Application.AxisDir``, which are: ``XNeg``, ``YNeg``,
-            ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``.  The default
-            is ``Application.AxisDir.XNeg``.
-        sourcename : str, optional
+        assignment : str or int or :class:`pyaedt.modeler.cad.object3d.Object3d`
+            First solid connected to the voltage source.
```

Comment:
```suggestion
            First object connected to the voltage source.
```
Was solid needed. It doesn't appear in the description for the next parameter. It seems that these two descriptions should be very similar.